dev/src/TestOutputChannel.ts

```diff
@@ -0,0 +1,34 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { OutputChannel } from "vscode";
+
+export class TestOutputChannel implements OutputChannel {
+    public name: string = 'Extension Test Output';
+
+    public append(value: string): void {
+        console.log(value);
```

Comment:
Technically this is wrong, but I guess it's good enough for now.

Reply:
I assume because of the new line?

Reply:
yeah